pom.xml

```diff
@@ -618,6 +618,26 @@
                 </execution>
               </executions>
             </plugin>
+            <plugin>
+              <artifactId>maven-javadoc-plugin</artifactId>
+              <version>3.0.1</version>
+              <configuration>
+                <additionalparam>-Xdoclint:none</additionalparam>
+                <sourceFileExcludes>
+                  <sourceFileExclude>**/target/generated-sources/**</sourceFileExclude>
+                </sourceFileExcludes>
+                <javadocExecutable>${java.home}/bin/javadoc</javadocExecutable>
```

Comment:
🤔 Afaict this shouldn't be necessary

Reply:
I will send you the internal stackoverflow explaining this.